sites/fast-website/src/app/style/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const siteBreakpoints: string[] = [
+    "0",
```

Comment:
The min breakpoint should probably be `320px` to indicate that it's the smallest size we want to support

Reply:
Makes sense.  Easy enough to change.